__main__.py

```diff
@@ -0,0 +1,211 @@
+import sys
+from cli import parse_args, validate_paths
+
+import pandas as pd
+import os
+
+import json
+from pathlib import Path
+
+def orginal_code(input_path, output_directory):
+    #!/usr/bin/env python
+    # coding: utf-8
+
+    # In[1]:
```

Comment:
What is this?  Is there a better way to clean this up?  Is this a TODO for later?